### PR TITLE
Updates since `0.8.0`

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: incomplete-cat-B-items.md [hunk 1]
```diff
@@ -13,10 +12,7 @@ This file has been auto-generated based on README.md. Do not edit it manually
 |  B  |          | 1.2.18    | wayland-protocol                | xdg-foreign-unstable-v2                                 |
```
Comment (Consolatis) [inline incomplete-cat-B-items.md:12]: xdg-foreign now supported as well

File: incomplete-cat-B-items.md [hunk 2]
```diff
@@ -13,10 +12,7 @@ This file has been auto-generated based on README.md. Do not edit it manually
...
 |  B  |          | 1.2.20    | wayland-protocol                | xwayland-keyboard-grab-unstable-v1                      |
 |  B  |          | 1.2.28    | wayland-protocol                | content-type-v1                                         |
-|  B  |          | 1.2.31    | wayland-protocol                |                                                         |
-|  B  |          | 1.2.32    | wayland-protocol                |                                                         |
```
Comment (Consolatis) [inline incomplete-cat-B-items.md:17]: What are/were these two?

File: incomplete-cat-B-items.md (hunk 1 above)
Comment (johanmalm) [reply incomplete-cat-B-items.md:12]: How could I forget :smile:
Thanks for catching. 
Have updated.

File: incomplete-cat-B-items.md (hunk 2 above)
Comment (johanmalm) [reply incomplete-cat-B-items.md:17]: I think they were just empty lines in preparation for adding more.
I probably got too hasty with `yyp` in vim at some point!!

**Pull request merged**